.cookiecutter/includes/README/head.md

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
Do we need to remember to have a leading space here? We could shift this to the parent

Reply:
I couldn't figure out a better way to do this, although that doesn't mean one doesn't exist. It's a battle with Jinja2's [whitespace control](https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control). You want the whitespace (number of blank lines before and after) in the final `README.md` file to be exactly right but when `.cookiecutter/includes/README/head.md` does exist and when it doesn't.

It would certainly be nicer if we didn't need to have this empty line here but I haven't been able to get it to work